packages/client/src/StreamEndpoints.ts

```diff
@@ -27,18 +27,10 @@ import { StreamRegistry } from './StreamRegistry'
 import { NodeRegistry } from './NodeRegistry'
 import { StreamIDBuilder } from './StreamIDBuilder'
 
-export interface StreamListQuery {
-    name?: string
-    uiChannel?: boolean
-    noConfig?: boolean
-    search?: string
-    sortBy?: string
+export interface SearchStreamsOptions {
```

Comment:
Should this interface definition be moved to StreamRegistry.ts?

Reply:
Moved